test/dist_autograd_test.py

```diff
@@ -1292,5 +1292,77 @@ def test_backwards_nested_python_udf(self):
             self.assertEqual(t1.grad, grads[t1])
             self.assertEqual(t2.grad, grads[t2])
 
+
+    _test_clean_context_backward_context_id = None
+
+    class MyBackwardFunc(Function):
+        @staticmethod
+        def forward(ctx, input):
+            return input
+
+        @staticmethod
+        @once_differentiable
+        def backward(ctx, input):
+            assert(DistAutogradTest._test_clean_context_backward_context_id is not None)
+
+            # Release the context to simulate error (use barrier before releasing context to ensure all nodes execute the backward function).
```

Comment:
comment line too long

Reply:
What is the convention for python line length in OSS? The python linter isn't complaining and I recall internally the line length is 150 chars.

Reply:
I am not aware if there is any convention here, I personally use 100. Feel free to ignore this comment though if all linters do not complain